src/test/java/com/wikia/webdriver/common/core/networktrafficinterceptor/NetworkTrafficInterceptor.java

```diff
@@ -55,6 +55,16 @@ public boolean searchRequestUrlInHar(String needle) {
     return false;
   }
 
+  public HarEntry getMatchingEntryByUrlPatternFromHar(String pattern) {
```

Comment:
`checkAssetsStatuses` works in a bit different way so I left it as it is. I've addressed the rest.